src/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project>
   <PropertyGroup>
-    <Version>1.13.12</Version>
+    <Version>1.13.13</Version>
```

Comment:
I think we don't need to do this version bump. We already have one version bump for v1.13.12 (the latest release was 1.13.11)